tools/bitfield_gen.py

```diff
@@ -1248,6 +1248,13 @@ def sign_extend_proof(high, base_bits, base_sign_extend):
         return ""
 
 
+def det_values(*dicts):
```

Comment:
If this becomes a perf issue, consider using a `SortedDict` (we already have sorted containers as a python dep) so you don't need to sort every time. http://www.grantjenks.com/docs/sortedcontainers/sorteddict.html